### DISCUSSION TITLE
CDC 관련 지표에는 무엇이 있을까?

### DISCUSSION THREAD

**Discussion by kyungjunleeme** [kyungjunleeme/kafka_study#40] [General]
aws 기준
https://docs.aws.amazon.com/ko_kr/dms/latest/userguide/CHAP_Monitoring.html#CHAP_Monitoring.Metrics.Task

**Comment by hellices** [comment]
kafka cdc를 오픈소스로 하게 되면 주로 debezium을 참고하게 됩니다.
debezium은 모니터링할 수 있는 지표를 각각 database별로 구성할 수 있도록 공개하고 있고, 이를 위한 대시보드도 제공하고 있습니다.
https://debezium.io/documentation/reference/3.0/connectors/oracle.html#oracle-snapshot-metrics
https://github.com/debezium/debezium-examples/tree/main/monitoring
